test/integration/docker-compose-migration.yml.template

```diff
@@ -6,7 +6,7 @@ services:
     image: {{ .DockerImage }}
     ports:
       - "{{with .ExposedRPCPorts}}{{index . 0}}:{{end}}8484"
-      - "{{with .ExposedRPCPorts}}{{index . 0}}:{{end}}8485"
+      - "{{with .ExposedRPCPorts}}{{index . 0 | plus 100}}:{{end}}8485"
```

Comment:
haha I don't see how this is different than the previous approach, other tests still need to change the port being used right?

Reply:
Nah, looks like it's not required, we always request the address based on the unexposed port which is always 8485.

Reply:
Oh my bad, I thought the CI failed the other day is due to admin test fail because of the port changes. Just checked the output, it's the 'pool overlap' issue - -